src/shared/tpm2-util.c

```diff
@@ -256,6 +256,25 @@ static bool tpm2_test_parms(Tpm2Context *c, TPMI_ALG_PUBLIC alg, const TPMU_PUBL
         return rc == TSS2_RC_SUCCESS;
 }
 
+static inline bool tpm2_supports_tpmt_sym_def_object(Tpm2Context *c, const TPMT_SYM_DEF_OBJECT *parameters) {
+        assert(parameters);
+
+        return tpm2_test_parms(c, TPM2_ALG_SYMCIPHER, &(const TPMU_PUBLIC_PARMS){ .symDetail.sym = *parameters, });
```

Comment:
we usually add some newlines when initialization a compound struct

Reply:
i created a local TPMU_PUBLIC_PARMS var so it's more readable